Tools/TestWebKitAPI/Tests/WebKitCocoa/AdditionalReadAccessAllowedURLs.mm

```diff
@@ -105,4 +113,83 @@
     TestWebKitAPI::Util::run(&done);
 }
 
+TEST(WebKit, NSAttributedStringWithTooManyReadOnlyPaths)
+{
+    __block bool done = false;
+
+    auto [readableDirectoryURL, unreadableDirectoryURL] = readableAndUnreadableDirectories();
+    NSURL *bundlePathURL = [NSURL fileURLWithPath:[[NSBundle mainBundle] bundlePath]];
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunguarded-availability-new"
+    NSMutableDictionary *options = [[NSMutableDictionary alloc] initWithObjectsAndKeys:@[ readableDirectoryURL, unreadableDirectoryURL, bundlePathURL ], WKReadAccessFileURLsOption, nil];
```

Comment:
The `NSMutableDictionary` is leaked here because you don't call `[options release]` later.  (I assume this is _not_ compiled with ARC.)

Use a `RetainPtr< NSMutableDictionary >` and `adoptNS()` to fix the leak.

Reply:
Ugh. Fixed!